config/swagger.yaml

```diff
@@ -2368,6 +2387,9 @@ definitions:
       fee:
         type: integer
         format: int64
+      ttl:
+        type: integer
+        format: int64
```

Comment:
This may impact (broken) code generators from swagger spec, though out Python client is the only one I am aware of.

Reply:
I think the swagger file would benefit from a normalization, the "same" things are expressed in a multitude of ways today.